frontend/src/routes/+layout.svelte

```diff
@@ -1,34 +1,38 @@
 <script lang="ts">
-  import { page } from '$app/stores';
-  import '$lib/app.postcss';
-  import { error, goesToErrorPage } from '$lib/error';
-  import UnexpectedErrorAlert from '$lib/error/UnexpectedErrorAlert.svelte';
-  import { onDestroy } from 'svelte';
-  import type { LayoutData } from './$types';
+import { page } from '$app/stores';
```

Comment:
Did your editor remove this indentation? I should probably figure out how to standardize more things.